react/components/Checkbox/index.js

```diff
@@ -46,13 +46,13 @@ class Checkbox extends PureComponent {
     return (
       <div
         className={classNames(
-          'vtex-checkbox__line-container flex items-center relative',
+          'vtex-checkbox__line-container flex items-start relative',
           {
             pointer: !disabled,
           }
         )}>
         <div
-          className={classNames('vtex-checkbox__container relative w1 h1', {
+          className={classNames(`vtex-checkbox__container relative w1 h1`, {
```

Comment:
So you can rollback this then 😅 
```suggestion
          className={classNames('vtex-checkbox__container relative w1 h1', {
```

Reply:
Done ✅